english/yoga/lifestyle/yoga_lifestyle.md

```diff
@@ -1,15 +1,15 @@
 Yoga lifestyle
 
 Yoga lifestyle is about one’s internal values: 
-What moves or motivates a person to do what they do? 
```

Comment:
I just removed some space at the end of the line in this file. Please review this, @casey-collab